### ISSUE TITLE
ti Inappropriate ioctl for device STDOUT

### ISSUE THREAD

**Issue by rekahsoft** [jeffWelling/ticgit#83]
Hi there, i installed ticgit-ng using "gem install TicGit-ng" but when i go to run "ti --help" i get:

/home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/lib/ticgit-ng/cli.rb:190:in `ioctl': Inappropriate ioctl for device - <STDOUT> (Errno::ENOTTY)
    from /home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/lib/ticgit-ng/cli.rb:190:in`try_using'
    from /home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/lib/ticgit-ng/cli.rb:179:in `reset_window_width'
    from /home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/lib/ticgit-ng/cli.rb:274:in`<top (required)>'
    from /usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
    from /usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in`require'
    from /home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/lib/ticgit-ng.rb:38:in `<top (required)>'
    from /usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in`require'
    from /usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
    from /home/collin/.gem/ruby/2.0.0/gems/TicGit-ng-1.0.2.17/bin/ti:11:in`<top (required)>'
    from /home/collin/.gem/ruby/2.0.0/bin/ti:23:in `load'
    from /home/collin/.gem/ruby/2.0.0/bin/ti:23:in`<main>'

**Comment by jeffWelling** [comment]
Hi there!
Can you tell me more about your environment? What Operating System, what distro, et al? I can see you're using Ruby 2.0.0, is that the version that you intended to use (re RVM -- are you using RVM?) or just the default invocation?
I'll see if I can replicate this if you can tell me which Distro/OS to replicate it on, thanks!

**Comment by nthorne** [comment]
Same issue here, running Linux Mint 16, ticgit installed via gem.

```
$ uname -a
Linux slaptopen 3.11.0-12-generic #19-Ubuntu SMP Wed Oct 9 16:12:00 UTC 2013 i686 i686 i686 GNU/Linux

$ ruby --version
ruby 1.9.3p194 (2012-04-20 revision 35410) [i686-linux]
```

**Comment by nthorne** [comment]
The `rescue no-tty error` pull request will resolve this issue.

**Comment by jeffWelling** [comment]
Merged and resolved in 1.0.2.18
